### DISCUSSION TITLE
How can I possibly limit the number of sessions for a given user or even all.

### DISCUSSION THREAD

**Discussion by drexhacker** [fastapi-users/fastapi-users#1374] [Q&A]
I am using `fastapi-users` with beanie using DatabaseBackend. So I am curious to know how I can limit the number of sessions a single user can have.

**Comment by frankie567** [comment]
You can probably overload the `write_token` method of the `DatabaseStrategy`:

https://github.com/fastapi-users/fastapi-users/blob/61dba2694c3c03dd0fca2a6407bf1f42c6726027/fastapi_users/authentication/strategy/db/strategy.py#L43-L46

For example, you can at this point check if the user has other tokens and remove them.
